go.sum

```diff
@@ -1,13 +1,12 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
Do we really need all these dependencies to be added?

Reply:
these were leftovers and are gone now. no new dependencies